addOns/imagelocationscanner/CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+- All rule has been tagged of interest to Penetration Testers, as well as adding tags associated with DEV or QA applicability.
```

Comment:
`The rule…`? `DEV` should be dropped, it was not.

Reply:
Done